projects/igniteui-angular/src/lib/slider/slider.component.ts

```diff
@@ -1122,28 +1124,26 @@ export class IgxSliderComponent implements
 
         if (this.isRange) {
             if (thumbType === SliderHandle.FROM) {
-                const newLower = this.lowerValue + value;
-                if (newLower >= this.lowerBound && newLower <= this.upperValue) {
-                    this._lowerValue = newLower;
-                    this.lowerValueChange.emit(this._lowerValue);
+                if (this.lowerValue + value > this.upperValue) {
+                    this.upperValue = this.lowerValue + value;
```

Comment:
Hm, I dont think this makes sense. It seems upperValue is restricted in range, so it make no sense to set it to `this.lowerValue + value`

I believe it was only the if statement in original code that was causing errors, and now in new code it is just enough to remove the if clause and just keeps this.lowerValuye += value and this.upperValue += value.

Reply:
I tried without adjusting the upper and lower values and it didn't work. The handle was getting stuck at the upper value when moving the lower handle beyond it, and at lower, when moving upper handle beyond it. I made the upper and lower adjust to the other handle if the interaction is still in the provided bounds.

Reply:
Thanks for explaining. I tested thoroughly once again and in my observation is that adjusting the `upperValue` and `lowerValue` in the if statement did not make difference. Still this is not so important anyway.

I confirm the problematic behavior is now gone and `lowerValue` and `upperValue` are updated as expected on lines 1130 and 1135.

Based on my testing, I logged another issue - https://github.com/IgniteUI/igniteui-angular/issues/13490 which may be planned as a minor enhancement for near sprints.